.github/workflows/sa-workflows.yml

```diff
@@ -8,6 +8,16 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+  repository_dispatch:
+    types: [ deploy_success ]
```

Comment:
Thanks @orangejenny ! Just 1 question, when the deploy will be success how what value will we get for the environment? I mean how to will it return values like banner, rogers, secure and securevoteu?

Reply:
Would that code go into this same file?

I was thinking the calling script (the cURL command) would include the environment as part of the JSON at sends when it triggers a `repository_dispatch` event.

Reply:
I am attaching the yml with the repository_dispatch and the matrix. This file follows the same pattern as that of the other ongoing QA scripts.
[sa-workflows.yml](https://github.com/user-attachments/files/22748234/sa-workflows.yml)
@orangejenny